### PR TITLE
feat(web-iconics): Add RTK Query to teams page

### PR DESCRIPTION
**Pull request by zerihunMoges**

### CODE REVIEW

**Comment by Aklile-Yilma** [comment]
LGTM

**Review by Aklile-Yilma — DISMISSED** [review]
LGTM

**Review by Aklile-Yilma — DISMISSED** [review]

File: Web/starter_project_iconics/components/teams/TeamMembersList.tsx [hunk 1]
```diff
@@ -1,31 +1,78 @@
...
+  }, [isLoading, isSuccess, startIndex, endIndex, teamMembers, allTeamMembers])
 
-        <div className="flex flex-col">
-            <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3  justify-items-center items-center gap-0 p-16">
-                {teamMembers.map((teamMember, index) => <TeamMemberCard key={index} teamMember={teamMember}/>)}
+  if (isLoading) {
+    content = (
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3  justify-items-center items-center gap-0 p-16">
+        {Array(3)
+          .fill(0)
```
Comment (Henok-Matheas) [inline Web/starter_project_iconics/components/teams/TeamMembersList.tsx:34]: use the length.

**Review by Henok-Matheas — APPROVED** [review]
LGTM

**Pull request merged**